tests/functional/adapter/incremental/test_incremental_unique_id.py

```diff
@@ -13,7 +13,8 @@
     config(
         materialized='incremental',
         incremental_strategy='merge',
-        unique_key=['state', 'county', 'city']
+        unique_key=['state', 'county', 'city'],
```

Comment:
@septimit, can you please explain why this is required "index='PRIMARY INDEX"?

Reply:
+1

Reply:
Tests fail against new TD instances if this definition is not there